types/react-netlify-form/index.d.ts

```diff
@@ -11,7 +11,7 @@ export interface NetlifyFormState {
     error: boolean;
     success: boolean;
     recaptchaError?: boolean | undefined;
-    recaptcha?: Recaptcha | undefined;
+    recaptcha?: React.ReactElement | undefined;
```

Comment:
Tests only passed because `recaptcha` was assignable to `{}` but it's an element: https://github.com/escaladesports/react-netlify-form/blob/f8008ace44d1c2bb9f27b199d36a3822d31367d3/src/export/index.js#L61-L68

Reply:
should this be handled in the types of `react-google-recaptcha` instead?

Reply:
Why should it be handled in another package if this package is responsible for the type at runtime?

Reply:
i guess i'm confused why `Recaptcha` doesn't work if (i think?) `Recaptcha` is the type that `react-google-recaptcha` defines

Reply:
`Recaptcha` is a component. The source link shows that `recaptcha` is the element created from `Recaptcha`.